google-cloud-datastore/pom.xml

```diff
@@ -38,6 +39,10 @@
       <groupId>com.google.cloud.datastore</groupId>
       <artifactId>datastore-v1-proto-client</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.auth</groupId>
```

Comment:
where is this added/used?

Reply:
One of my earlier builds had failed with an error for missing `com.google.auth` dependency - so I'd added it then. Maybe my mvn repo wasn't correctly set up. Don't need this anymore.

Reply:
Nvm. The kokoro build did fail. https://github.com/googleapis/java-datastore/actions/runs/9006313814/job/24743556228 I'm going to reinstate the dependency.